src/lib/storage/vector_compression/bitpacking/bitpacking_vector_type.hpp

```diff
@@ -5,7 +5,15 @@
 
 namespace opossum {
 
-// Right now, compact_vector is only used in the BitpackingVector where the value type uint32_t is sufficient.
+/**
+ * @brief Template arguments of compact::vector<typename IDX, unsigned BITS, typename W, typename Allocator>
+ *
+ * IDX: right now, compact_vector is only used in the BitpackingVector where the value type uint32_t is sufficient.
+ * BITS: 0 means that the bit width is determined at runtime
+ * W: the compact::vector allocates its memory in chunks of the specified word size.
```

Comment:
Not that "wasting" 32bit in some cases should be an issue, but is there a good reason to use `uint64_t` here?

Reply:
it is the default value that they use. I could also use `uint32_t` but I don't think it really makes any difference, so I'd just go with the default.